components/centraldashboard/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compilerOptions":{
         "module":"commonjs",
         "esModuleInterop":true,
-        "target":"es6",
+        "target":"es2019",
```

Comment:
Why is this necessary? Is it because of the `Object` methods being used? Could we implement in a way that doesn't require more recent JS support? It doesn't really matter for the server side, but it's a big deal for the browser.

Reply:
`Object.fromEntries()` and `includes()` aren't in es6 but are both in es2019